tanner/emulators/xxe_injection.py

```diff
@@ -55,8 +56,10 @@ def scan(self, value):
 
         result = await self.get_injection_result(attack_params[0]['value'])
         if not result or 'stdout' not in result:
+            self.logger.exception('Error while getting the injection results from php sandbox..')
```

Comment:
show the output of sandbox?

Reply:
Output of sandbox is not neat, a lot of random stuff can be there. I dont think it will be good?

Reply:
Yeah, right